6_visualize/src/prep_datetime_fun.R

```diff
@@ -0,0 +1,9 @@
+
+prep_datetime_fun <- function(datetime){
+  datetime <- strftime(as.POSIXct(datetime), format = '%b %d %I:%M %p')
+
+  plot_fun <- function(){
+    plot(datetime, add = TRUE, col = 'grey')
```

Comment:
Looks like the right format for this file overall, but can you just call plot on a character string? I'm trying it locally and keep getting this error:
```
Error in plot.window(...) : need finite 'ylim' values
In addition: Warning messages:
1: In xy.coords(x, y, xlabel, ylabel, log) : NAs introduced by coercion
2: In min(x) : no non-missing arguments to min; returning Inf
3: In max(x) : no non-missing arguments to max; returning -Inf
4: In plot.window(...) : "add" is not a graphical parameter
```

Reply:
I think you want `text`, right? And to be able to position it in the right spot. I have something similar in the `prep_legend` if that is helpful.